connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorConfigTest.java

```diff
@@ -119,4 +120,41 @@ public void testNonMutationOfConfigDef() {
             connectorConfigDef.names().contains(taskSpecificProperty)
         ));
     }
+
+    @Test
+    public void testSourceConsumerConfig() {
```

Comment:
We should test settings with the `source.cluster.` prefix too. Same in the test below